tasks/27_basic-subclasses/basic-subclasses.js

```diff
@@ -0,0 +1,28 @@
+class God {
+  static create() {
+    const man = new Man('Adam', 'male');
+    const woman = new Woman('Eve', 'female');
+    return [man, woman];
+  }
+}
+
+class Human {
+  constructor(name) {
+    this.name = name;
+  }
+}
+
+class Man extends Human {
+  constructor(name, sex) {
+    super(name);
+    this.sex = 'male';
+  }
+}
+class Woman extends Human {
+  constructor(name, sex) {
+    super(name);
+    this.sex = 'female';
```

Comment:
Why do you define `this.sex` in `Woman` constructor instead of declaring it in Human and passing it there